services/rfq/relayer/service/handlers.go

```
@@ -91,32 +91,43 @@
+		dbReq.Reason = fmt.Sprintf("invalid token (no contract code at address)")
+		err = r.db.StoreQuoteRequest(ctx, dbReq)
+		if err != nil {
+			return fmt.Errorf("could not store db request: %w", err)
+		}
 		return nil
 	}
-
 	if err != nil || originDecimals == nil || destDecimals == nil {
-		return fmt.Errorf("could not get decimals: %w", err)
+		dbReq.Status = reldb.WillNotProcess
+		dbReq.Reason = fmt.Sprintf("could not get decimals: %w", err)
```

golangci-lint (1.62.2)

printf: fmt.Sprintf does not support error-wrapping directive %w

(govet)